### PR TITLE
vrops-exporter: added diskspace alerts for vcenter and esxi

### PR DESCRIPTION
**Pull request by christopherhans**

### CODE REVIEW

File: prometheus-exporters/vrops-exporter/alerts/host.alerts [hunk 1]
```diff
@@ -66,3 +66,27 @@ groups:
     annotations:
       description: "Failover Host {{ $labels.hostsystem }} has Virtual Machines on it. Free up the host. ({{ $labels.vcenter }}, {{ $labels.vccluster }})"
       summary: "Failover Host {{ $labels.hostsystem }} has Virtual Machines on it. ({{ $labels.vcenter }}, {{ $labels.vccluster }})"
+  - alert: HostDiskspaceUsageWarning
```
Comment (richardtief) [inline prometheus-exporters/vrops-exporter/alerts/host.alerts:69]: I would prefer `DiskSpace` with an uppercase S.

File: prometheus-exporters/vrops-exporter/alerts/host.alerts [hunk 2]
```diff
@@ -66,3 +66,27 @@ groups:
...
+    expr: vrops_hostsystem_diskspace_usage_gigabytes / vrops_hostsystem_diskspace_capacity_gigabytes > 0.7
+    for: 30m
+    labels:
+      severity: info
+      tier: vmware
+      service: compute
+      context: "vrops-exporter"
+      meta: "Diskpace usage of host {{ $labels.hostsystem }} is above 70%. ({{ $labels.vcenter }}, {{ $labels.vccluster }})"
```
Comment (richardtief) [inline prometheus-exporters/vrops-exporter/alerts/host.alerts:77]: Missing `s` ind `Diskspace`.

Comment (richardtief) [reply prometheus-exporters/vrops-exporter/alerts/host.alerts:77]: Copy paste error occurs multiple times.

**Review by richardtief — CHANGES_REQUESTED** [review]
Looks good. However, I made some comments.

File: prometheus-exporters/vrops-exporter/alerts/host.alerts (hunk 1 above)
Comment (richardtief) [reply prometheus-exporters/vrops-exporter/alerts/host.alerts:69]: Also for the other alerts.

Comment (christopherhans) [reply prometheus-exporters/vrops-exporter/alerts/host.alerts:69]: yeah, you are right. It is disk space, not diskspace

**Review by viennaa — APPROVED** [review]

**Pull request merged**